README.md

```diff
@@ -4,5 +4,24 @@ _Extracts articles from [Wikipedia database dumps](https://en.wikipedia.org/wiki
 
 ## Usage
 
+To serve as a drop-in replacement for the descriptions scraper:
+- Install this tool to `$PATH` as `om-wikiparser`.
```

Comment:
It doesn't need to be, the example script read more clearly to me if it's in the context of the `intermediate_data` directory. It could also be run as `../../../wikiparser/target/release/om-wikiparser`, with `cargo run --release` from the wikiparser directory, or anything else.

Reply:
...then why suggesting to install the tool at PATH?

Reply:
1. Can it be wrapped in a helper script that can be easily customized and run on the generator, maybe directly from the wikiparser repo? :)
2. `cargo run -r` may be even better instead of a path to binary :) But it's also ok to hard-code the path or use `$WIKIPARSER_BINARY` var.

Think about me testing your code soon on a production server. Less surprises = less stress ;-)

Reply:
I will update the README to be more of an explanation, and make another issue/PR for a script that handles the build directory, timing, backtraces, saving logs, etc.